dbus-wait.sh

```diff
@@ -8,7 +8,10 @@ wait_for_dbus() {
 			  /org/freedesktop/DBus \
 			  org.freedesktop.DBus.ListNames
 
-		if [ $? ]; then
```

Comment:
Here's an example that shows why I think this did not work:

```
➜  cat fail.sh
false
if [ $? ]; then
	echo "LAST TREATED AS SUCCESS: But it failed"
else
	echo "LAST TREATED AS FAILURE: That's right"
fi

➜  sh ./fail.sh 
LAST TREATED AS SUCCESS: But it failed
➜  cat work.sh 
false
if [ $? -ne 0 ]; then
	echo "LAST TREATED AS FAILURE: That's right"
else
	echo "LAST TREATED AS SUCCESS: But it failed"
fi
➜  sh ./work.sh 
LAST TREATED AS FAILURE: That's right
```

Reply:
Aha so adding -ne 0 should fix it?